games-engines/scummvm/scummvm-1.9.0.recipe

```diff
@@ -56,41 +58,56 @@ BUILD_REQUIRES="
 	devel:libogg$secondaryArchSuffix
 	devel:libpng$secondaryArchSuffix
 	devel:libSDL$secondaryArchSuffix
+	#devel:libsdl_net$secondaryArchSuffix #not needed yet
 	devel:libtheora$secondaryArchSuffix
 	devel:libvorbis$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-"
+	"
 
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
-	cmd:git
 	cmd:ld$secondaryArchSuffix
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:nasm
 	cmd:pkg_config$secondaryArchSuffix
-"
+	"
 
 BUILD()
 {
 	libtoolize --force --copy --install
 	CPPFLAGS=`freetype-config --cflags` ./configure --prefix=$prefix \
-	--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
-	--mandir=$manDir --docdir=$docDir \
-	--disable-debug --enable-release #--disable-all-engines
-	#--disable-all-engines can be used to only compile scummvm without
-	#engines, which makes it build faster, those that are still under
-	#construction can be built with --enable-all-engines for testing purposes
-
+		--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
+		--mandir=$manDir --docdir=$docDir \
+		--disable-debug --enable-release --disable-eventrecorder \
+		#--disable-all-engines #can be used to compile scummvm without \
+		#engines, which makes it build faster (to test), those that \
+		#are still under construction can be built with --enable-all-engines \
+		#for testing purposes
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 	mv $appsDir/scummvm $appsDir/ScummVM
-	addResourcesToBinaries $portDir/additional-files/scummvm.rdef \
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/scummvm.rdef > scummvm.rdef
```

Comment:
I think that adding a **`.in`** suffix to the template rdef file might be a good idea.
Of course, you would need to change the last line of the sed command to:
```
		$portDir/additional-files/scummvm.rdef.in > scummvm.rdef
```
and update:
```
ADDITIONAL_FILES="scummvm.rdef.in"
```
For your convenience, here are the steps to rename the rdef file:
```
cd /path/to/haikuports/games-engines/scummvm/additional-files
git mv scummvm.rdef scummvm.rdef.in
git commit --amend scummvm.rdef scummvm.rdef.in
```
What about a last tiny change:
```
HOMEPAGE="https://www.scummvm.org/"
```
(Please don't forget the trailing slash ;-)
Thanks!